pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -82,9 +120,10 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    license = "GPL";
-    homepage = https://www.inkscape.org;
     description = "Vector graphics editor";
+    homepage = "https://www.inkscape.org";
+    license = licenses.gpl3Plus;
+    maintainers = [ maintainers.jtojnar ];
```

Comment:
Could you please not backport the maintainers change? I am explicitly not maintaining software on stable releases: https://github.com/NixOS/nixpkgs/commit/986799d95f56b8c511bb33eae12e9337b40d1b8e

Reply:
Done. Thanks.